INSTALL.md

```diff
@@ -1,222 +1,73 @@
 # Installing smoke-alarm-portal
 
-These installation instructions assume a Debian GNU/Linux 'testing'
-distribution operating system, but they should be portable without
-much difficulty to most other Unix-like operating systems.
+These instructions assume you have [Docker](https://docs.docker.com/install/) installed.
```

Comment:
```suggestion
These instructions assume you have [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
```

Reply:
The twilio library is upset that no SID and auth token were passed to it. You can fill those with dummy values in .env to get this to run

Reply:
Hmm, since it looks like this will be used largely for development, I think it's fine to put dummy values in here with comments specifying as much.